packages/manager/apps/web/client/app/domain/anycast/translations/Messages_fr_FR.json

```
@@ -1,3 +1,17 @@
 {
-  "domain_anycast": "Activer DNS Anycast"
+  "domain_anycast": "Activer DNS Anycast",
+  "domain_dashboard_dns_anycast_order": "Commander DNS Anycast",
+  "domain_configuration_anycast_order_description": "Activation de l'option DNS Anycast sur le domaine {{t0}}",
+  "domain_dashboard_dns_anycast_payment": "Paiement",
+  "domain_dashboard_dns_anycast_duration_choose": "Choisissez votre durée",
+  "domain_dashboard_dns_anycast_price_with_tax": "{{ price }} TTC",
+  "domain_dashboard_dns_anycast_price_without_tax": "{{ price }} HT",
+  "domain_dashboard_dns_anycast_next": "Suivant",
+  "domain_dashboard_dns_anycast_duration": "12 mois",
+  "domain_dashboard_dns_anycast_pay": "Payer",
+  "domain_configuration_anycast_order_success": "Votre demande d'activation de l'option DNS Anycast a bien été prise en compte.",
+  "domain_configuration_anycast_order_success_bc": "Afin de finaliser celle-ci, merci de bien vouloir procéder au réglement de votre bon de commande.",
+  "domain_configuration_anycast_order_success_bc_link": "Bon de commande",
+  "domain_configuration_anycast_payment_acceptance_delay_info": "Veuillez noter qu'un délai d'environ 15mn à compter de l'acceptation de votre paiement est nécessaire pour que l'activation de l'option soit prise en compte dans votre Espace Client Web.",
```

`mn` is an outdate shortcut usage of minutes, we have to use `min`.

```suggestion
  "domain_configuration_anycast_payment_acceptance_delay_info": "Veuillez noter qu'un délai d'environ 15min à compter de l'acceptation de votre paiement est nécessaire pour que l'activation de l'option soit prise en compte dans votre Espace Client Web.",
```